src/transformStateWithClones.js

```diff
@@ -7,7 +7,57 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const NewArray = [];
+  let massive = { ...state };
+
+  for (let numArr = 0; numArr < actions.length; numArr++) {
```

Comment:
I recommend using `for...of` loop if you iterate over all elements of the array - it's faster